tests/events/test_utils.py

```diff
@@ -129,6 +185,68 @@ def test_content(self):
             },
         )
 
+        # Create events get nothing redacted.
```

Comment:
```suggestion
        # After MSC2176, create events get nothing redacted.
```

For consistency with the similar comment below.